source/codegen/metadata/niscope_restricted/functions.py

```diff
@@ -131,4 +131,118 @@
         ],
         'returns': 'ViStatus'
     },
+    'GetStartTimestampInformationWithChannels': {
```

Comment:
@Avinash2Suresh , FYI when you get to generating the restricted scope service metadata from hapigen you should include these methods too.